packages/apollo-gateway/src/index.ts

```diff
@@ -236,8 +242,13 @@ export class ApolloGateway implements GraphQLService {
     }
 
     if (isRemoteConfig(config)) {
+      const serviceList = config.serviceList.map(serviceDefinition => ({
+        serviceDefinition,
```

Comment:
Nit: I'd be interested in flattening this structure into `{ name, url, dataSource }`.